third_party/txt/src/txt/paragraph_txt.cc

```diff
@@ -1105,6 +1105,12 @@ void ParagraphTxt::Layout(double width) {
         if (style.has_height_override) {
           // Scale the ascent and descent such that the sum of ascent and
           // descent is `fontsize * style.height * style.font_size`.
+          //
+          // The raw metrics do not add up to fontSize. The state of font
+          // metrics is a mess, see
```

Comment:
What is the relationship between the content on that page and what libtxt is doing?  I'd prefer to explain what the code is doing here versus citing a web page containing general information.

Reply:
+1 links come and go, and sometimes change. Just add a brief summary of the salient bits from that page as they pertain to this code.